Bottom: 47bf03b751f72dac2da23b880118c9b54e1cf4c5
Top:    7e4fdf10c960a283cb8df1e0804f2a0d52e375f7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 02:18:16 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
